### PR TITLE
Replace <see langword="UInt64" /> tags

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: xml/System.Runtime.Serialization/SerializationInfo.xml [hunk 1]
```diff
@@ -1043,7 +1043,7 @@ End Sub
       </Parameters>
       <Docs>
         <param name="name">The name to associate with the value, so it can be deserialized later.</param>
-        <param name="value">The <see langword="UInt64" /> value to serialize.</param>
+        <param name="value">The <see cref="T:System.UInt64" /> value to serialize.</param>
```
Comment (bartonjs) [inline xml/System.Runtime.Serialization/SerializationInfo.xml]: If I interpret the [style guidelines](https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors) correctly, the type just shouldn't even be here.

```suggestion
        <param name="value">The value to serialize.</param>
```

But I agree the change as proposed is an improvement.

Comment (mairaw) [reply xml/System.Runtime.Serialization/SerializationInfo.xml]: Yep, that's a good suggestion @bartonjs.

**Comment by mairaw** [comment]
Feel free to ping anyone from the @dotnet/docs team from now on @joperezr since I moved to the PM team, but happy to help if there are questions.

**Comment by joperezr** [comment]
Sorry for that @mairaw. @dotnet/docs @BillWagner @tdykstra @Thraka any pushback with getting this in?

**Pull request merged**